src/components/Stepper/Step.tsx

```diff
@@ -161,15 +161,29 @@ export const Step: React.FC<StepProps> = ({
         )}
         justify="center"
       >
-        <Icon
-          aria-hidden="true"
-          className={clsx(
-            classes.icon,
-            active && classes.activeIcon,
-            completed && classes.completedIcon
-          )}
-          role="img"
-        />
+        {Icon ? (
+          <Icon
+            aria-hidden="true"
+            className={clsx(
+              classes.icon,
+              active && classes.activeIcon,
+              completed && classes.completedIcon
+            )}
+            role="img"
+          />
+        ) : (
+          <Text
+            aria-hidden="true"
```

Comment:
@ynotdraw should we have `aria-hidden` here? Since there is no title or subtitle text for the step, shouldn't we want to allow screen readers to pick up this text?

Reply:
Hm to be honest, I'm not quite sure.  I'd have to do some reading on how steppers are expected to work when it comes to a11y. It's _maybe_ okay as-is?

I'll file a task to revisit a11y for this component.

Reply:
Sweet, thanks!

Reply:
@HaleyWardo @ynotdraw Thank you both for taking a look!